website/docs/networking/vpc-lattice/traditionalabtesting.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Traditional A/B testing"
```

Comment:
This is not technically A/B testing, but just deploying a new container. Instead, we could deploy two version of the checkout container with the same labels. This would mean the checkout service would route to both versions of the checkout app, making for rudimentary A/B testing. 2 checkout v1 pods and 2 checkout v2 pods behind the same service should mean 50% to each. Obviously this is not ideal, and sets up nicely for the next step where you can setup granular rules for routing.

Reply:
As mentioned above, I tried but this makes the setup (and cleanup) very complex due to the way the application is built. I don't have have a strong preference for this, however this makes it a lot easier.